testlist/other.dat

```diff
@@ -0,0 +1,28 @@
+# We do not have a toolchain for avr32 outside of Microchip login wall.
+# The work was never upstreamed to GCC.
+
+/avr
+-avr32dev1:nsh
+-avr32dev1:ostest
+
+/renesas/rx65n/rx65n-grrose
+/renesas/rx65n/rx65n-rsk2mb
+-Darwin,rx65n.*
+
+# PINGUINOL toolchain doesn't provide macOS binaries
+# with the same name
+/mips,CONFIG_MIPS32_TOOLCHAIN_PINGUINOL
+-Darwin,pic32.*
```

Comment:
All the configs under boards/mips/ need to be blocked.
```
mirtoo
pic32mx7mmb
pic32mx-starterkit
sure-pic32mx
ubw32
flipnclick-pic32mz
pic32mz-starterkit
```

Reply:
Done.